### ISSUE TITLE
DJIWindowsSDKSample is unloaded

### ISSUE THREAD

**Issue by Tabinda12** [dji-sdk/Windows-SDK#66]
DJIWindowsSDKSample is unloaded update required problem is found.
![error_@](https://user-images.githubusercontent.com/75783922/101759169-134dcc00-3afb-11eb-93d6-4170b13ae493.PNG)

**Comment by dji-dev** [comment]
Agent comment from William Wong in <a href='https://djisdksupport.zendesk.com/agent/tickets/41950'>Zendesk ticket #41950</a>:<br><br>Dear Client
Thank you for contacting DJI.

I think it is probably your visual studio version, are you using VS 2017 wtih Windwos 10 SDK 16299 and 17134?

Hopefully our solution can help you.
Kindly Regards,
DJI Developer Support

**Comment by Tabinda12** [comment]
Yes i m using Visual Studio 2107 along with Windows SDK version 17134.

**Comment by Tabinda12** [comment]
Sorry for the late previous email.
I m using Visual Studio 2019 and Windows SDK 17134. Is it necessary to use
Visual Studio 2017?.
Guide me ps.
Thank you.
